controllers/replication.storage/volumereplication_controller.go

```diff
@@ -363,13 +364,42 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 
 	instance.Status.LastCompletionTime = getCurrentTime()
+
+	var requeueForInfo bool
+	rawScheduleTime := parameters["schedulingInterval"]
+	if rawScheduleTime == "" {
+		rawScheduleTime = "1m"
+	}
+	scheduleTime, err := time.ParseDuration(rawScheduleTime)
+	if err != nil {
+		logger.Error(err, "failed to parse time: %v", rawScheduleTime)
+		return ctrl.Result{}, err
+	}
```

Comment:
```suggestion
       const defaultScheduleTime = time.Minute

        func getScheduleTime(parameters map[string]string) (time.Duration) {
		rawScheduleTime := parameters["schedulingInterval"]
		if rawScheduleTime == "" {
			return defaultScheduleTime
		}
		scheduleTime, err := time.ParseDuration(rawScheduleTime)
		if err != nil {
			logger.Error(err, "failed to parse time: %v", rawScheduleTime)
			return defaultScheduleTime
		}
		return scheduleTime
	}
```

Create a constant and a function like above and use it . 

@yati1998 ^